app/autocomplete_mention/routes.test.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+var expect = require('chai').expect
+  , request = require('supertest');
+
+var app = require('../../app');
+
+describe('Autocomplete', function() {
+  describe('GET /mention', function() {
+
+    before(function(done) { // add users to user collection in db
+      var users = [
+        { name : 'world' },
+        { name : 'work' },
+        { name : 'woman' }
+      ];
+      require('../../lib/mongo').connect(function(err,db) {
+        db.collection('user').drop( // but clear collection first
```

Comment:
@sscotth There is a special case where a person has already added `{ data : items }` to the `'user'` collection prior to running the test, in which case there may be, for example, `{ name : 'worldly' }`.  In this case, the test will fail and will require being run twice to pass.